centos-stream-8-build.sh

```diff
@@ -302,92 +318,96 @@ echo "Building containers-common..."
+    # Podman container configs
     install -vDm 644 mounts.conf -t "$pkgdir/etc/containers/"
 
     (
-        tar -xf "common-$pkgver.tar.gz"
-        cd "common-$pkgver"
+        tar -xf "common-$pkgver.tar.gz" \
+          && cd "common-$pkgver"
+
         # configs
         install -vDm 644 pkg/config/containers.conf -t "$pkgdir/etc/containers/"
         install -vDm 644 pkg/config/containers.conf -t "$pkgdir/usr/share/containers/"
         install -vDm 644 pkg/seccomp/seccomp.json -t "$pkgdir/etc/containers/"
         install -vDm 644 pkg/seccomp/seccomp.json -t "$pkgdir/usr/share/containers/"
     )
-    
+
     (
-        tar -xf "image-$_image_pkgver.tar.gz"
-        cd "image-$_image_pkgver"
+        tar -xf "image-$_image_pkgver.tar.gz" \
+          && cd "image-$_image_pkgver"
+
         # configs
         install -vDm 644 registries.conf -t "$pkgdir/etc/containers/"
     )
 
     (
-        tar -xf "shortnames-$_shortnames_pkgver.tar.gz"
-        cd "shortnames-$_shortnames_pkgver"
+        tar -xf "shortnames-$_shortnames_pkgver.tar.gz" \
+          && cd "shortnames-$_shortnames_pkgver"
+
+        # configs
         install -vDm 644 shortnames.conf "$pkgdir/etc/containers/registries.conf.d/00-shortnames.conf"
     )
 
     (
-        tar -xf "skopeo-$_skopeo_pkgver.tar.gz"
-        cd "skopeo-$_skopeo_pkgver"
+        tar -xf "skopeo-$_skopeo_pkgver.tar.gz" \
+          && cd "skopeo-$_skopeo_pkgver"
+
         # configs
         install -vDm 644 default-policy.json "$pkgdir/etc/containers/policy.json"
         install -vDm 644 default.yaml -t "$pkgdir/etc/containers/registries.d/"
     )
 
     (
-        tar -xf "storage-$_storage_pkgver.tar.gz"
-        cd "storage-$_storage_pkgver"
+        tar -xf "storage-$_storage_pkgver.tar.gz" \
+          && cd "storage-$_storage_pkgver"
+
         # configs
         install -vDm 644 storage.conf -t "$pkgdir/etc/containers/"
         install -vDm 644 storage.conf -t "$pkgdir/usr/share/containers/"
     )
 )
 echo "Building containers-common done."
 cp_to_real_pkgdir
-
 ldconfig
 
 # crun
 # https://github.com/archlinux/svntogit-community/blob/packages/crun/trunk/PKGBUILD
 echo "Building crun..."
 (
     pkgname=crun
-    pkgver=1.6
+    pkgver=1.7.2
 
-    curl -sSLo crun.static "https://github.com/containers/crun/releases/download/${pkgver}/${pkgname}-${pkgver}-linux-amd64"
-    echo "69ba74a05cc6147cf65e1bebb79e2adb8c96012eb3a701f5faa51869251cb2dc crun.static" > crun.static.sha256
-    sha256sum -c crun.static.sha256
-    chmod +x crun.static
+    curl -sSLo "$pkgname-static" "https://github.com/containers/crun/releases/download/${pkgver}/${pkgname}-${pkgver}-linux-amd64"
+    echo "2bd2640d43bc78be598e0e09dd5bb11631973fc79829c1b738b9a1d73fdc7997 crun-static" > "$pkgname-static.sha256"
+    sha256sum -c "$pkgname-static.sha256"
+    chmod +x "$pkgname-static"
     mkdir -p "$pkgdir/usr/bin"
-    cp crun.static "$pkgdir/usr/bin/"
+    cp "$pkgname-static" "$pkgdir/usr/bin/crun-static"
```

Comment:
You've renamed `crun.static` to `crun-static` here but it's still patched as `crun.static` in the config file in the `entrypoint.sh`.